worlds/raft/__init__.py

```diff
@@ -43,16 +43,23 @@ class RaftWorld(World):
     data_version = 2
     required_client_version = (0, 3, 4)
 
+    raft_frequencyItemsPerPlayer = {}
```

Comment:
This variable has the potential to bleed through multiple generations.

Reply:
Good point. Any world that uses this will set the value it's interested in using, so at worst this will keep some stale references to empty arrays around. If there's a method that should be used for cleanup to reset this after a multiworld is generated (or just after Raft is finished using this, looks like `post_fill()` would work but might not be the "correct" location) I can add that in.

Reply:
You could create and place them immidiatly, so you have no need to track them in the first place.
Alternatively, you can attach the data to self.multiworld instead, which is one generation instance.

Reply:
Thanks for the recommendations. I've moved to using an object in `self.multiworld`.